build.sh

```diff
@@ -6,11 +6,31 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 PROJECT_DIR=$DIR/
 
 # This will: compile the project, run lint, run tests under JVM, package apk, check the code quality and run tests on the device/emulator.
-"$PROJECT_DIR"/gradlew --no-daemon clean
-"$PROJECT_DIR"/gradlew --no-daemon build -Dscan
-"$PROJECT_DIR"/gradlew --no-daemon test
-"$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=small
-if [ "$TRAVIS_BRANCH" == "development" ] || [ "$TRAVIS_BRANCH" == "master" ];
-then "$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=medium; fi
-if [ "$TRAVIS_BRANCH" == "master" ];
-then "$PROJECT_DIR"/gradlew --no-daemon connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.size=large; fi
\ No newline at end of file
+if [ "$TEST_SUITE" == "units" ];
+then
+"$PROJECT_DIR"/gradlew clean
+"$PROJECT_DIR"/gradlew build -Dscan
+"$PROJECT_DIR"/gradlew test; fi
```

Comment:
/gradlew test is not necessary because this task is executed with /gradlew build -Dscan instruction